_data/icons.yaml

```diff
@@ -0,0 +1,50 @@
+- name: highlight-off
```

Comment:
This and the following files were ripped from [live site](https://github.com/canonical/design.ubuntu.com/tree/main/_data)